abstract_specification.md

```diff
@@ -49,78 +45,78 @@ class DataFrame():
         """
 ```
 
-If a class, method or parameter is not explicitly marked with a lifecycle stage, it is assumed to be `experimental`.
+If a class, method, or parameter is not explicitly marked with a lifecycle stage, it is assumed to be `experimental`.
 
 # Data Model
 
-The data model is comprised of two layers:
+The data model comprises two layers:
 
-- a set of "foundational" types which are general in nature
-- a set of "composed" types, which are composed from the foundational types, and are intended to improve ease of dataset use and interoperability
+* a set of _foundational_ types which are general in nature;
+* a set of _composed_ types, which are composed from the foundational types and are intended to improve ease of dataset use and interoperability.
 
 The foundational types are:
 
-- SOMACollection - a string-keyed container (key-value map) of other SOMA data types, e.g., SOMADataFrame, SOMADataMatrix and SOMACollection.
-- SOMADataFrame - a multi-column table -- essentially a dataframe with indexing on user-specified columns.
-- SOMADenseNDArray and SOMASparseNDArray- an offset addressed (zero-based), single-type N-D array, available in either sparse or dense instantiations
+* `SOMACollection`: a string-keyed container (key-value map) of other SOMA data types, e.g. `SOMADataFrame`, `SOMASparseNDArray`, and `SOMACollection`.
+* `SOMADataFrame`: a multi-column table -- essentially a dataframe with indexing on user-specified columns.
+* `SOMADenseNDArray` and `SOMASparseNDArray`: an offset-addressed (zero-based), single-type N-D array, available in either sparse or dense instantiations.
 
 The composed types are:
 
-- SOMAExperiment - a specialization and extension of SOMACollection, codifying a set of naming and indexing conventions to represent annotated, 2-D matrix of observations across _multiple_ sets of variables.
+* `SOMAExperiment`: a specialization and extension of `SOMACollection`, codifying a set of naming and indexing conventions to represent an annotated, 2-D matrix of observations across _multiple_ sets of variables.
```

Comment:
Measurement also belongs here, though that doesn't need to be handled in this change.

Reply:
@thetorpedodog tracked here:
https://github.com/single-cell-data/SOMA/issues/133